oss-workshop.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Open Source Workshop
+---
+
+4/26/15 12-5pm, [Projective Space LES](http://www.projective.co/), 72 Allen St. 3rd Floor, NYC
```

Comment:
I put 12-5 somewhat arbitrarily – thoughts on a good amount of time, assuming we will have some orientation and maybe little demos of what people worked on at the end?